Makefile

```diff
@@ -15,7 +15,7 @@ python:
 		-t /local/templates/python \
 		-l python \
 		-c /local/config/python.json \
-		--additional-properties 'infoEmail=hello@seldon.io,pyMinVersion=3.6' \
+		--additional-properties 'infoEmail=hello@seldon.io,pyMinVersion=3.6,pyMaxVersion=3.9' \
```

Comment:
**question:** I'm not 100% sure if this is the correct way. This `additional-properties` is a Swagger thing. I'm verifying how this shows up atm.

Reply:
**update:** After a rebuild (`make python`) there are no differences.

Reply:
If you feel like it, you could also tweak the `./templates/python/setup.py` to take into account that `pyMaxVersion` property into the actual package - so that `pip` verifies the Python version at installation time.

Reply:
✅ I had to bump `setuptools` [1]

[1] https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires